src/client/src/components/pages/participants/addParticipant/index.jsx

```diff
@@ -37,36 +37,36 @@ export default class index extends Component {
       confirmButtonAriaLabel: "Thumbs up",
       cancelButtonText: '<i className="fa fa-thumbs-down"></i> No ',
       cancelButtonAriaLabel: "Thumbs down"
-    }).then(confirm => {
-      if (confirm.value) {
-        axios("/api/v2/participants", {
-          method: "POST",
-          data: {
-            participantdata: obj
-          }
-        }).then(result => {
-          if (result.data.error) {
-            swal({
-              title: "",
-              type: "warning",
-              html: result.data.error,
-              confirmButtonText: "Ok"
-            });
-          } else {
-            swal({
-              title: "Success",
-              type: "success",
-              html: result.data.message
-            }).then(confirm => {
-              if (confirm.value) {
-                this.setState({ ...obj });
-                this.props.history.push("/participants/view");
-              }
-            });
-          }
+    });
+    if (confirm.value) {
+      const result = await axios("/api/v2/participants", {
+        method: "POST",
+        data: {
+          participantdata: obj
+        }
+      });
+      if (result.data.error) {
+        const warning = await swal({
+          title: "",
+          type: "warning",
+          html: result.data.error,
+          confirmButtonText: "Ok"
+        });
+        if (warning.value) {
+          this.props.history.push("/participants/view");
+        }
+      } else {
+        const success = await swal({
+          title: "Success",
+          type: "success",
+          html: result.data.message
         });
+        if (success.value) {
+          this.setState({ ...obj });
+          this.props.history.push("/participants/view");
+        }
```

Comment:
Cool!  If you like it, you could use it for addCourse above too.

Reply:
Haha, you just did

Reply:
I used it :smile: